lib/fontist/font.rb

```diff
@@ -96,11 +96,11 @@ def formula
 
     def downloadable_font
       if formula
-        raise(
-          Fontist::Errors::MissingFontError,
+        raise Fontist::Errors::MissingFontError.new(
```

Comment:
Should just pass the "name" to the error and have the Error class build the message.

Reply:
Now it's much easier to throw these errors :)